INSTALL.md

```diff
@@ -64,7 +69,16 @@ See [instructions](./docker/README.md)
 	},
 ```
 
-5. Now that you have built the binaries, follow instructions in the _Running a Standalone Server_ section.
+5. If you bundling all available DB adapters in the Tinode binary, make sure you specify the adapter name in your `tinode.conf`. E.g. you want to run Tinode with MySQL:
```

Comment:
`all available DB adapters` -> `more than one DB adapter`

Reply:
Changed the wording. `use_adapter` must always be present in `tinode.conf`.